package/MDAnalysis/lib/util.py

```diff
@@ -1402,25 +1459,54 @@ def get_weights(atoms, weights):
 #: translation table for 3-letter codes --> 1-letter codes
 #: .. SeeAlso:: :data:`alternative_inverse_aa_codes`
 canonical_inverse_aa_codes = {
-    'ALA': 'A', 'CYS': 'C', 'ASP': 'D', 'GLU': 'E',
-    'PHE': 'F', 'GLY': 'G', 'HIS': 'H', 'ILE': 'I',
-    'LYS': 'K', 'LEU': 'L', 'MET': 'M', 'ASN': 'N',
-    'PRO': 'P', 'GLN': 'Q', 'ARG': 'R', 'SER': 'S',
-    'THR': 'T', 'VAL': 'V', 'TRP': 'W', 'TYR': 'Y'}
+    "ALA": "A",
+    "CYS": "C",
+    "ASP": "D",
+    "GLU": "E",
+    "PHE": "F",
+    "GLY": "G",
+    "HIS": "H",
+    "ILE": "I",
+    "LYS": "K",
+    "LEU": "L",
+    "MET": "M",
+    "ASN": "N",
+    "PRO": "P",
+    "GLN": "Q",
+    "ARG": "R",
+    "SER": "S",
+    "THR": "T",
+    "VAL": "V",
+    "TRP": "W",
+    "TYR": "Y",
+}
 #: translation table for 1-letter codes --> *canonical* 3-letter codes.
 #: The table is used for :func:`convert_aa_code`.
-amino_acid_codes = {one: three for three,
-                    one in canonical_inverse_aa_codes.items()}
+amino_acid_codes = {
+    one: three for three, one in canonical_inverse_aa_codes.items()
+}
 #: non-default charge state amino acids or special charge state descriptions
 #: (Not fully synchronized with :class:`MDAnalysis.core.selection.ProteinSelection`.)
 alternative_inverse_aa_codes = {
-    'HISA': 'H', 'HISB': 'H', 'HSE': 'H', 'HSD': 'H', 'HID': 'H', 'HIE': 'H', 'HIS1': 'H',
-    'HIS2': 'H',
-    'ASPH': 'D', 'ASH': 'D',
-    'GLUH': 'E', 'GLH': 'E',
-    'LYSH': 'K', 'LYN': 'K',
-    'ARGN': 'R',
-    'CYSH': 'C', 'CYS1': 'C', 'CYS2': 'C'}
```

Comment:
This is the first one where I think the old formatting makes more sense so we should keep it. I'll add a guard.

Reply:
They are kinda still ordered either way, I'd be happy with the new version too!

Reply:
Yeah, I think here it makes sense to disable formatting.